gateway-ha/src/main/java/io/trino/gateway/ha/HaGatewayLauncher.java

```diff
@@ -6,22 +6,27 @@
 import io.trino.gateway.baseapp.BaseApp;
 import io.trino.gateway.ha.config.HaGatewayConfiguration;
 
-public class HaGatewayLauncher extends BaseApp<HaGatewayConfiguration> {
+public class HaGatewayLauncher
+        extends BaseApp<HaGatewayConfiguration>
+{
+    public HaGatewayLauncher(String... basePackages)
+    {
+        super(basePackages);
+    }
 
-  public HaGatewayLauncher(String... basePackages) {
-    super(basePackages);
-  }
+    public static void main(String[] args)
```

Comment:
The original bottom position looks good to me.

Reply:
I am not sure what you mean .. is there anything I should change now?

Reply:
I meant moving this main method to the bottom.

Reply:
Intellij reorder I guess .. will move down